src/axom/quest/MarchingCubes.hpp

```diff
@@ -151,11 +151,13 @@ class MarchingCubes
   /*!
    \brief Computes the isocontour.
    \param [in] contourVal isocontour value
+   \param [in] maskVal mask value.  If a mask field is given in
+     setMesh(), compute only for cells whose mask matches this value.
 
    Each computeIsocontour call adds to previously computed contour
    mesh.
   */
-  void computeIsocontour(double contourVal = 0.0);
+  void computeIsocontour(double contourVal = 0.0, int maskVal = 1);
```

Comment:
Would it be better to have a `setMaskValue` instead of passing in `maskVal` here?

Reply:
Since the mask field is provided outside this call, and masking seems somewhat non-conventional for MC, I'm leaning towards this being set outside the `computeIsoContour` function. I.e. users who don't care about masking won't have to think about it when calling `computeIsocontour`

Presumably, the following would work equally well:
```cpp
for(auto v : maskValues)
{
   mc.setMaskValue(v);
   mc.computeIsocontour(iso);
}
```

Reply:
Thank you.  Implemented.